packages/coordinator/src/middlewares/default/block-proposer.ts

```diff
@@ -98,15 +98,15 @@ export class BlockProposer extends ProposerBase {
       this.context.account.getChainId(),
       this.context.account.getTransactionCount('latest'),
     ])
-    const signedTx = await this.context.account.signTransaction({
+    const tx = await this.context.account.sendTransaction({
       ...proposeTx,
       chainId,
       nonce,
       gasLimit: expectedGas,
       maxFeePerGas: this.context.effectiveGasPrice,
-      type: 2,
+      // FIXME: add it back
+      // type: 2,
```

Comment:
remove it for testing bcs our hardhat doesn't support `type == 2`(EIP1559)

Reply:
Would you make this only for testnet?
the Zkopru testnet on Goerli, the coordinator works well with type-2 tx, and it is good for the user gas fee (even if testnet though )

Reply:
sure, it's just a workaround for now and will fix it later.